infra/dev/docker-compose.yml

```diff
@@ -84,8 +84,8 @@ services:
       timeout: 1s
       retries: 30
 
-  geoserver-stubs:
-    container_name: monitorfish_geoserver_stubs
+  geoserver-monitorenv-stubs:
```

Comment:
I did that because it's logical to have a root docker-compose that describes all the services that are available and can be run on a repository, and then extend from that everywhere else we need it. It's for self-documentation and clarity purposes.

For example, running the Cypress-dedicated docker-compose for local development is king of strange, at least to me.